optionals/compat_rhs_usf3/CfgMagazineWells.hpp

```diff
@@ -0,0 +1,8 @@
+class CfgMagazineWells {
+    class ace_hellfire_K {
+        GVAR[] = {QGVAR(pylon_mag_2rnd_hellfire_k)};
```

Comment:
Actually.... GVAR[] ?
Does that work? Shouldn't work in my mind.
`ADDON` seems better?